docs/manual/src/udl/errors.md

```diff
@@ -42,3 +42,85 @@ interface ArithmeticError {
   IntegerOverflow(u64 a, u64 b);
 };
 ```
+
+## Interfaces as errors
+
+** This is not avaiable in all bindings **
+
+It's possible to use an `interface` (ie, a rust struct impl) as an error;
```

Comment:
I'm not sure if "rust struct impl" is the right term for this, but I don't have anything much better.  Maybe just linking to the interface documents?